src/ops.jl

```diff
@@ -189,7 +184,7 @@ trace(A::FixedMatrix{2,2}) = A[1,1] + A[2,2]
 trace(A::FixedMatrix{3,3}) = A[1,1] + A[2,2] + A[3,3]
 trace(A::FixedMatrix{4,4}) = A[1,1] + A[2,2] + A[3,3] + A[4,4]
 
-\{m,n,T}(mat::Mat{m,n,T}, v::Vec{n, T}) = inv(mat)*v
+\{m,n,T1,T2}(mat::Mat{m,n,T1}, v::Vec{n,T2}) = inv(mat)*v
```

Comment:
Huh.  Interesting paper thanks!

I'd expect the straight-line code we use for `inv` for tiny matrices to be very fast.  Presumably there's a cross over point for not very large matrices where it's better to just create a `Matrix` and use backslash instead.